public/schema.json

```diff
@@ -11,8 +11,8 @@
   },
   "tags": [
     {
-      "name": "Subjects",
-      "description": "Subject endpoints"
+      "name": "Study Events",
```

Comment:
I meant there should be two tags here:
Subject & Study Events depending on what the endpoint relates to